gslib/tests/signurl_signatures.py

```diff
@@ -67,3 +67,21 @@
                                    'ential=test%40developer.gserviceaccount.com%2F19000101%2Fasia%'
                                    '2Fstorage%2Fgoog4_request&x-goog-date=19000101T000555Z&x-goog-'
                                    'expires=0&x-goog-signedheaders=host')
+
+TEST_SIGN_URL_GET_PY3 = "https://storage.googleapis.com/test/test.txt?x-goog-signature=b'5d2c2179fe191b95f52a8df108c5d71ade3c758cdc56cac4409012629bfbc9f0343a473bd8fc98cb05b6510e2e05a89c3a410dd2b6dc0ca48546221bec9b0a4d73a16c45eb3e2ea115482caf5547ca24c1e388d33a59be583b730f268998a47b9b284b25a66d611564f8b56c7cad1eea4049e8838d8d8cfc85c1d2d073f513118446b59bfd7f20bc59786e1034a8d7c4d565fb2deeb89ffea1eaa535f9cb87e78eb87c479cdda95f23551b8b7c6974b82105210806d7285c5361c4203192dbf12de7ac376681ee72cc460ab6828eef3b8ca2e6ac5590f925f7497bc3fb15d3e4a1d8084c6b02b09e1be96070c776511669bfa47a6e250b429e2298c05470a9d5'&x-goog-algorithm=GOOG4-RSA-SHA256&x-goog-credential=test%40developer.gserviceaccount.com%2F19000101%2Fasia%2Fstorage%2Fgoog4_request&x-goog-date=19000101T000555Z&x-goog-expires=0&x-goog-signedheaders=host"
```

Comment:
All the hashes ending in `_PY*` can be removed. They were consolidated in a previous PR review